### PR TITLE
[NUI][Tizen.NUI.PerformanceTest] Fix reference bug

### PR DESCRIPTION
**Pull request by hinohie**
Due to the Animation reference become zero, GC dispose it.
So It didn't call the OnAnimationFinished callback before disposed.
That's why deleteCount not matched with tickCount.

This patch fix that bug

Signed-off-by: Eunki, Hong <eunkiki.hong@samsung.com>

### CODE REVIEW

File: test/Tizen.NUI.PerformanceTest/PerformanceTest.cs [hunk 1]
```diff
@@ -475,7 +488,7 @@ protected override void OnCreate()
     [STAThread] // Forces app to use one thread to access NUI
     static void Main(string[] args)
     {
-        PerfNewExample example = new PerfNewExample();
+        PerformanceTestExample example = new PerformanceTestExample();
         example.Run(args);
     }
 }
```
Comment (dongsug-song) [inline test/Tizen.NUI.PerformanceTest/PerformanceTest.cs:494]: this is trivial thing but from next time, could you add new line at the end of file?
reference: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: test/Tizen.NUI.PerformanceTest/tizen-manifest.xml [hunk 2]
```diff
@@ -1,16 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns="http://tizen.org/ns/packages" api-version="6" package="org.tizen.example.Tizen.NUI.PerfNew" version="1.0.0">
+<manifest xmlns="http://tizen.org/ns/packages" api-version="6" package="org.tizen.example.Tizen.NUI.PerformanceTest" version="1.0.0">
   <profile name="common" />
-  <ui-application appid="org.tizen.example.Tizen.NUI.PerfNew"
-					exec="PerfNew.dll"
+  <ui-application appid="org.tizen.example.Tizen.NUI.PerformanceTest"
+					exec="PerformanceTest.dll"
 					type="dotnet"
```
Comment (dongsug-song) [inline test/Tizen.NUI.PerformanceTest/tizen-manifest.xml:6]: from next time, please use "dotnet-nui" to activate "preloading"
thank you.

**Review by dongsug-song — APPROVED** [review]
looks good to me

File: test/Tizen.NUI.PerformanceTest/tizen-manifest.xml (hunk 2 above)
Comment (hinohie) [reply test/Tizen.NUI.PerformanceTest/tizen-manifest.xml]: Fix done. thank you.

File: test/Tizen.NUI.PerformanceTest/PerformanceTest.cs (hunk 1 above)
Comment (hinohie) [reply test/Tizen.NUI.PerformanceTest/PerformanceTest.cs]: Fix done. thank you.

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Pull request merged**